app/locales/ms/translations.js

```diff
@@ -469,7 +469,7 @@ export default {
                         "label": "Saya memberi tumpuan atau bekerja keras pada tugasan."
                     },
                     "9": {
-                        "label": "Saya menyimpan dan tidak meluahkan."
+                        "label": "Saya memendam (reserved) dan tidak meluahkan perasaan"
```

Comment:
Why does the malaysian translation contain a word of english? This is marked as a production translation, which means we may want to resolve this question before shipping this release.